app/editor/src/features/admin/actions/styled/ActionForm.tsx

```diff
@@ -8,6 +8,5 @@ export const ActionForm = styled(FormPage)`
 
   .back-button {
     align-self: start;
-    margin-bottom: 3%;
```

Comment:
Removed this from most places as it was a wasting vertical space.  @jdtoombs was this required for some reason?